pkg/controller.v2/tensorflow/controller.go

```diff
@@ -279,9 +284,6 @@ func (tc *TFController) syncTFJob(key string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	if len(namespace) == 0 || len(name) == 0 {
-		return false, fmt.Errorf("invalid tfjob key %q: either namespace or name is missing", key)
```

Comment:
Actually, I am not sure why we could remove the validation here.

Reply:
func SplitMetaNamespaceKey will check it. And we can not create/update tfjob metadata let tfjob's namespace or name be empty